docs/extras/advanced-variables/variable-defaults.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Variable Defaults
+sidebar_position: 2
+---
+
+Variable defaults are the values you provide in code which tell DevCycle SDKs what value to assign to a Variable in
+cases where a different value is not available. 
+
+It's worth understanding which scenarios a Variable will be defaulted in, as this can differ from other Feature Flagging
+platforms.
+
+## When is the default value used?
+
+### Configuration Not Available
+If the SDK has not yet received a configuration from DevCycle, it will return the default value for a Variable 
+as defined in code. 
+
+SDKs will attempt to retrieve the configuration until an error is received or a timeout is reached. In these cases,
+the configuration will not be obtained and default values will be used for the duration of the SDK instance's lifetime.
+
+### Type Mismatches
+In many SDKs, the type of the default value (Boolean, String etc.) determines the "expected" type of the Variable.
+On the DevCycle platform, a Variable is configured to have a specific type, and a Feature controls the values of that
+given type to assign to the Variable.
+
+But what if the type as defined in DevCycle does not agree with what your code expects? In that scenario, DevCycle
+will defer to the code's expected type by preserving the default value and not attempt to override it with the value
+from the configuration. This behaviour ensures that the SDK will not cause runtime errors due to type mismatches.
+
+For example, if a Variable is being evaluated in code and the Boolean value `false` is provided as a default value,
+the SDK will understand that that Variable should always be a Boolean. If the DevCycle configuration serves a String 
+value instead, the SDK will ignore it and use the default value of `false` instead.
+
+### Variable Does Not Exist
+In cases where the Variable does not exist in the DevCycle platform, the SDK will use the default value provided in code.
+
+### User Does Not Qualify for Targeting, or Feature is Turned Off
```

Comment:
...or Targeting is Turned Off

Reply:
i mentioned targeting in the paragraph below